pkg/openstack/ca.go

```diff
@@ -130,6 +131,19 @@ func ReconcileCAs(ctx context.Context, instance *corev1.OpenStackControlPlane, h
 			if err != nil {
 				return ctrl.Result{}, err
 			}
+			err = caOnlyBundle.getCertsFromPEM(caCert)
+			if err != nil {
+				return ctrl.Result{}, err
+			}
+
+			status := corev1.TLSCAStatus{
+				Name: caName,
+			}
+			if len(caOnlyBundle.certs) > 0 {
+				status.Expires = caOnlyBundle.certs[0].expire.String()
```

Comment:
Is this used anywhere else? Is it just so the use can see the expire time from the status?